sandbox/libs/authn/src/main/java/org/opensearch/authn/Subject.java

```diff
@@ -1,9 +1,12 @@
 /*
- * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
+ *
```

Comment:
Oh no! inflating licenses!

Reply:
Thanks! I updated to the shorter license for all the classes in this PR.